metrics/metricstest/metrics_mock.go

```diff
@@ -71,7 +72,7 @@ func (m *MockMetrics) RecordPutTotal() {
 	MockCounters["puts.current_url.request.total"] = MockCounters["puts.current_url.request.total"] + 1
 }
 func (m *MockMetrics) RecordPutDuration(duration time.Duration) {
-	MockHistograms["puts.current_url.duration"] = duration.Seconds()
+	MockHistograms["puts.current_url.duration"] = duration.Seconds() + 1.00
```

Comment:
Needed to explicitly add time in these mock `RecordXXXDuration` functions because the Github actions test was so fast that `duration.Seconds()` evaluated to zero and a false positive was being thrown.

Reply:
Modified to use mock duration instead